modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelper.java

```diff
@@ -473,7 +475,16 @@ protected void validateField(
 		builder.withInstanceId(fieldInstanceId);
 
 		if (!valid) {
-			String errorMessage = ddmFormFieldValidation.getErrorMessage();
+			LocalizedValue errorMessageLocalizedValue =
+				ddmFormFieldValidation.getErrorMessage();
+
+			String errorMessage = StringPool.BLANK;
```

Comment:
This line makes the code of line 489 unreachable.

Reply:
Sure, I fixed this part.